infra/relayer/src/relayer.js

```diff
@@ -281,10 +281,20 @@ class Relayer {
           logger.info(`Confirmed tx with hash ${hash}. Paid ${gas} gas`)
         })
       } catch (reason) {
+        let status = enums.RelayerTxnStatuses.Failed
+        let errMsg = 'Error sending transaction'
+
+        if (reason.message && reason.message.indexOf('gas prices') > -1) {
+          status = enums.RelayerTxnStatuses.GasLimit
+          errMsg = 'Network is too congested right now.  Try again later.'
+        }
+
         if (dbTx) {
-          await dbTx.update({ status: enums.RelayerTxnStatuses.Failed })
+          await dbTx.update({ status })
```

Comment:
I wonder if we should store the reason as a field in the data JSONB column ? For ease of troubleshooting rather than having to look into the machine logs.

Reply:
What's the purpose of the `data` field, anyway?  Doesn't appear used. Perhaps this is a TODO item?

Reply:
Data field was for future needs to store any interesting metadata. I think storing the details of the failure reason in there would be a good use for it.